ConsoleApp/Shop/Buyer.cs

```diff
@@ -0,0 +1,7 @@
+namespace ConsoleApp.Shop;
+using System.Collections.Generic;
+
+class Buyer
+{
+    private string _name { get; set; }
```

Comment:
not critical now, but for future - properties naming is always PascalCase
so here it should be
```suggestion
    private string Name { get; set; }
```